core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -2606,3 +2606,19 @@ secret_configs:
   - cluster: build02
     name: dex-rh-sso
     namespace: openshift-config
+- from:
```

Comment:
Shouldnt we also remove the copying of this into the secret that was used by the job?

Reply:
Do you mean this?
https://github.com/openshift/release/pull/15596

Reply:
Oh. I see.
This has to be done with another PR to let the current PR to pass the presubmits.
https://github.com/openshift/release/pull/15640